stochatreat/stochatreat.py

```diff
@@ -159,6 +154,12 @@ def stochatreat(data: pd.DataFrame,
         # get the number of misfits
         n_misfit = int(block_size - n_belong)
 
+        # to avoid bias towards the first treatment
+        rand_indices = list(range(treats))
```

Comment:
Why are we suffling indices instead of just filling an array with treatment ids and shuffling that? Do we need the indices still?

Reply:
to simultaneously shuffle both arrays the same!